source/appController/AMDSClientAppController.h

```diff
@@ -73,13 +75,16 @@ class AMDSClientAppController : public AMDSAppController
 signals:
 	/// signal to indicate that the manager object is opening a network session
 	void networkSessionOpening();
-	/// signal to indicate that a network session is opened
-	void networkSessionOpened();
 
-	/// signal to indicate that a new server is connected
-	void newServerConnected(const QString &hostIdentifier);
+	/// signal to indicate that the AMDS Client App Controller is ready for service
+	void AMDSClientControllerReady();
```

Comment:
I think we should distinguish between server closed and disconnected.  If you have taken the action to disconnect from the server then I don't think the server should emit an error.  If it is disconnected via other means, then it should definitely be an error.